sky/execution.py

```diff
@@ -474,14 +491,18 @@ def launch(
                     check_cloud_vm_ray_backend=False,
                     dryrun=dryrun)
                 handle = maybe_handle
-                # Get all stages
+                logger.info('provision')
```

Comment:
Do we need this or should we elaborate the message?

Reply:
oops